Automattic/jetpack
# [AI image] fix styling  issues

Fixes #28323 

## Proposed changes:
* Fix textarea width

### Other information:

- [x] Fix textarea width
- [ ] ~Make textarea height change depending on input text ( see [this comment](https://github.com/Automattic/jetpack/pull/28339#issuecomment-1383910122))~ **created a separate task for it**
- [x] Fix result images overflow 
- [x] Fix result images hovering

## Testing instructions:

* Deploy to your sandbox (with `jetpack-dowbnloader` for example)
* Check that all changes were made

## Screenshots

<img width="533" alt="Screenshot 2023-01-17 at 12 11 40" src="https://user-images.githubusercontent.com/790558/212884426-3ef8074f-3302-4815-8f5f-5cda5494b5a3.png">
<img width="689" alt="Screenshot 2023-01-17 at 12 07 41" src="https://user-images.githubusercontent.com/790558/212883716-54114a07-2404-4d75-af57-c516e645ff23.png">
<img width="606" alt="Screenshot 2023-01-17 at 12 07 45" src="https://user-images.githubusercontent.com/790558/212883732-d64b7bd2-8f52-4f8c-9088-94e9a959e794.png">
<img width="505" alt="Screenshot 2023-01-17 at 12 08 04" src="https://user-images.githubusercontent.com/790558/212883744-3a5dec78-97f7-471a-b931-a5bc816c69e8.png">
